src/components/IconInCircularCont.js

```diff
@@ -0,0 +1,35 @@
+/* @flow */
+import React, { Component } from "react";
+import { StyleSheet, View } from "react-native";
+
+type Props = {
+  icon: any,
+  bgIconContainer: string,
+  size: number,
+};
+class IconInCircularCont extends Component<Props> {
```

Comment:
we can probably call it `Circle` and rename icon props to `children` to write code like `<Circle size..><Icon />`
also maybe rename `bgIconContainer` to `bg`